src/main/java/io/github/railroad/utility/Actions.java

```diff
@@ -0,0 +1,71 @@
+package io.github.railroad.utility;
+
+import javafx.stage.FileChooser;
+import javafx.stage.FileChooser.ExtensionFilter;
+import javafx.stage.Modality;
+import org.apache.commons.io.FilenameUtils;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import static io.github.railroad.utility.Components.StageFactory.makeStage;
+
+public interface Actions {
```

Comment:
Bit of a weird name for this class, idk what other methods are supposed to be here though

Reply:
I think it's fine.